empty/src/client.ts

```diff
@@ -0,0 +1,40 @@
+// @@@SNIPSTART typescript-hello-client
+import { Connection, Client } from '@temporalio/client';
+import { nanoid } from 'nanoid';
+import { TASK_QUEUE_NAME } from './shared';
+
+// import your workflow
+import { YOUR_WORKFLOW } from './workflows';
+
+async function run() {
+  // Connect to the default Server location
+  const connection = await Connection.connect({ address: 'localhost:7233' });
+  // In production, pass options to configure TLS and other settings:
+  // {
+  //   address: 'foo.bar.tmprl.cloud',
+  //   tls: {}
+  // }
+
+  const client = new Client({
+    connection,
+    // namespace: 'foo.bar', // connects to 'default' namespace if not specified
+  });
+
+  const handle = await client.workflow.start(YOUR_WORKFLOW, {
+    taskQueue: TASK_QUEUE_NAME,
+    // type inference works! args: [name: string]
+    args: ['Temporal'],
+    // in practice, use a meaningful business ID, like customerId or transactionId
+    workflowId: 'workflow-' + nanoid(),
+  });
+  console.log(`Started workflow ${handle.workflowId}`);
+
+  // optional: wait for client result
+  console.log(await handle.result()); // Hello, Temporal!
```

Comment:
Maybe remove 'Hello, Temporal!' because we don't know that the result of the Workflow is going to be.